google-ads/build.gradle

```diff
@@ -68,8 +68,7 @@ dependencies {
     testImplementation 'com.google.api:gax-grpc::testlib'
     testImplementation 'io.grpc:grpc-context'
     testImplementation 'com.google.truth:truth:0.27'
-    implementation 'com.google.auto.value:auto-value-annotations:1.10.4'
-    testImplementation 'com.google.auto.value:auto-value-annotations:1.10.4'
+    implementation 'com.google.auto.value:auto-value-annotations'
```

Comment:
With this change, the version of `auto-value-annotations` will be dictated by `google-cloud-shared-dependencies`